tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -23,7 +22,7 @@ sub reboot_and_wait_up {
     if (is_s390x) {
         record_info('INFO', 'Reboot LPAR');
         #Reboot s390x lpar
-        power_action('reboot', observe => 1, keepconsole => 1);
+        enter_cmd "reboot";
```

Comment:
Seems it is okay for PASS jobs. How about failing jobs at reboot? Will you lose some debug log for reboot if directly typing reboot?

Reply:
So you have console log for the reboot process, right?

Reply:
No yet, but we have some ways to confirm our expected and required debug logs after reboot our s390x LPAR - journalctl or supportconfig. 

The current OSD s390x test run using the svirt BACKEND did not generate enough console log.